Dojo_class.py

```diff
@@ -0,0 +1,121 @@
+import random
+from Room_Class import *
+from Person_Class import *
+
+
+class Dojo(object):
+    def __init__(self):
+        self.all_rooms = []
+        self.livingspaces = []
+        self.offices = []
+        self.all_people = []
+        self.fellows = []
+        self.staff = []
+        self.available_offices = []
+        self.available_livingspaces = []
+        self.unallocated_offices = []
+        self.unallocated_livingspaces = []
+
+    def create_room(self, room_type, rooms_to_make):
+        if len(rooms_to_make) != 0:
+            for room_name in rooms_to_make:
+                if room_type == "office":
+                    office_instance = Office("office", room_name)           # to ensure the instance being worked on
+                    self.all_rooms.append(office_instance)                  # during the function call is singular
+                    self.offices.append(office_instance)
+                    self.available_offices.append(office_instance)
+                    print("An {} called {} has been created".format(office_instance.room_type, office_instance.room_name))
+                elif room_type == "livingspace":
+                    livingspace_instance = LivingSpace("livingspace", room_name)
+                    self.all_rooms.append(livingspace_instance)
+                    self.livingspaces.append(livingspace_instance)
+                    self.available_livingspaces.append(livingspace_instance)
+                    print("A {} called {} has been created".format(livingspace_instance.room_type, livingspace_instance.room_name))
```

Comment:
Limit length of lines to 72 characters.
https://www.python.org/dev/peps/pep-0008/